app/src/main/java/com/github/se/cyrcle/ui/tutorial/TutorialScreen.kt

```diff
@@ -0,0 +1,202 @@
+          ClickableText(
+              text =
+                  buildAnnotatedString {
+                    withStyle(style = SpanStyle(color = Color.Black)) {
+                      append(stringResource(R.string.tutorial_screen_button_skip))
+                    }
+                  },
+              modifier =
+                  Modifier.align(Alignment.End)
+                      .padding(end = generalPadding, bottom = generalPadding)
+                      .testTag("TutorialScreenSkipButton"),
+              style =
+                  TextStyle(
+                      textDecoration = TextDecoration.Underline, fontStyle = FontStyle.Italic),
+              onClick = { navigationActions.navigateTo(finalDestination) })
+        }
+  }
+}
+
+/** The Welcome screen of the tutorial, with some text to explain the content of the tutorial. */
+@Composable
+private fun TutorialWelcomeScreen() {
+  Spacer(Modifier.height(titleSpacer))
+
+  Text(
+      text = stringResource(R.string.tutorial_screen_welcome_title),
+      style =
+          MaterialTheme.typography.headlineLarge.copy(
+              fontWeight = FontWeight.SemiBold, fontSize = 45.sp, lineHeight = 64.sp),
+      modifier = Modifier.padding(generalPadding),
+      testTag = "TutorialScreenWelcomeTitle")
+
+  Spacer(Modifier.height(subtitleSpacer))
+
+  Text(
+      text = stringResource(R.string.tutorial_screen_welcome_subtitle),
+      style = MaterialTheme.typography.bodyMedium.copy(fontSize = 15.sp, lineHeight = 20.sp),
+      modifier = Modifier.padding(generalPadding),
+      testTag = "TutorialScreenWelcomeSubtitle")
+}
+
+/**
+ * Sub screen that displays an tutorial image.
+ *
+ * @param imageId The id (string ressource) of the image to display.
+ * @param contentDescription The content description of the image.
+ * @param testTag The test tag of the image.
+ */
+@Composable
+fun ImageSubScreen(imageId: Int, contentDescription: String, testTag: String) {
+  Image(
+      painter = painterResource(id = imageId),
+      contentDescription = contentDescription,
+      modifier = Modifier.testTag(testTag))
+}
+
+/**
+ * The Thank you screen of the tutorial. Contains some text to thank the user for completing the
+ * tutorial.
+ */
+@Composable
+fun ThankYouScreen() {
+  Spacer(Modifier.height(titleSpacer))
+
+  Text(
+      text = stringResource(R.string.tutorial_screen_thank_you_title),
+      style =
+          MaterialTheme.typography.headlineLarge.copy(
+              fontWeight = FontWeight.SemiBold, fontSize = 45.sp, lineHeight = 20.sp),
+      modifier = Modifier.padding(generalPadding),
+      testTag = "TutorialScreenThankYouTitle")
+
+  Spacer(Modifier.height(subtitleSpacer))
+
+  Text(
+      text = stringResource(R.string.tutorial_screen_thank_you_subtitle),
+      style = MaterialTheme.typography.bodyMedium,
+      modifier = Modifier.padding(generalPadding),
+      testTag = "TutorialScreenThankYouSubtitle")
+}
+
+// Uncomment to have a preview of the Tutorial
```

Comment:
It says uncomment in this comment, but in your PR it's uncommented already. Normal?

Reply:
I think it was meant to be commented regarding the code coverage
I left it because it is hard to see the tutorial from a phone in which you've already have an account (before the button in profile).
As there is now a button, I will remove this.